lib/gooddata/lcm/actions/collect_ca_metrics.rb

```diff
@@ -13,7 +13,13 @@ class CollectComputedAttributeMetrics < BaseAction
 
       PARAMS = define_params(self) do
         description 'Client Used for Connecting to GD'
-        param :gdc_gd_client, instance_of(Type::GdClientType), required: true
+        param :gdc_gd_client, instance_of(Type::GdClientType), required: false
+
+        description 'Development Client Used for Connecting to GD'
+        param :development_client, instance_of(Type::GdClientType), required: true
+
+        description 'Synchronization Info'
+        param :synchronize, array_of(instance_of(Type::SynchronizationInfoType)), required: true, generated: true
```

Comment:
is this really required to be an array of synchronizationInfoType? in the past implementation, if the params are an empty array, it would run correctly, which would not happen now